manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -97,7 +97,10 @@ sidecarInjectorWebhook:
     autoInject: true
 
   rewriteAppHTTPProbe: true
-
+istiodRemote:
```

Comment:
probably shouldn't have been scoped under `istiodRemote` originally... oh well.

Reply:
agree.. Let's address this later